f8a_worker/models.py

```diff
@@ -411,3 +411,35 @@ class OSIORegisteredRepos(Base):
     git_sha = Column(String(255), nullable=False)
     email_ids = Column(String(255), nullable=False)
     last_scanned_at = Column(DateTime)
+
+
+class ProbableCVEs(Base):
+    """Table for storing daily Probable CVEs"""
+
+    __tablename__ = "probable_cves"
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    ecosystem = Column(String(50), nullable=False)
+    repository = Column(String(100), nullable=False)
+    repo_url = Column(String(500), nullable=False)
+    package = Column(String(500), nullable=False)
+    cause_type = Column(String(50), nullable=False)
+    issue_date = Column(DateTime, nullable=True)
+    issue_url = Column(JSONB, nullable=True)
```

Comment:
Why JSONB?

Reply:
And it is used on other places as well.

Reply:
There can be more than one issues/prs/commits - that can cause a probable CVE.  And the model will send it as list/json.  We will save it as JSON in the table and show as comma separated list in the UI.  And based on what I read so far, JSONB is better managed internally by postgres than JSON.